src/main/java/org/folio/consortia/domain/entity/TenantEntity.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.consortia.domain.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Objects;
+
+@Getter
+@Setter
```

Comment:
You can use just lombok @Data annotanion and remove Getter, Setter, ToString, RequiredArgsConstructor as well as equals and hashcode